From 7f97369bc797253fe4a233cc0528858fbb28cfd0 Mon Sep 17 00:00:00 2001
From: akruphi <92621645+akruphi@users.noreply.github.com>
Date: Wed, 8 Jan 2025 17:22:18 +0300
Subject: [PATCH] Revert default of Size column to 6

Yet more after #2452

Current (after #2452) default 9 (or 0) make show files size as long string
 and it show not usual, for example, files with size 7198741 show all in bytes
 after revert to 6 it show as in classic view as "7030 K".
---
 far2l/src/panels/flmodes.cpp | 26 +++++++++++++-------------
 1 file changed, 13 insertions(+), 13 deletions(-)

diff --git a/far2l/src/panels/flmodes.cpp b/far2l/src/panels/flmodes.cpp
index 32900c07f..046da083c 100644
--- a/far2l/src/panels/flmodes.cpp
+++ b/far2l/src/panels/flmodes.cpp
@@ -87,51 +87,51 @@ PanelViewSettings ViewSettingsArray[] = {
 #endif
 
 PanelViewSettings ViewSettingsArray[] = {
-		/* 00 */ {{COLUMN_MARK | NAME_COLUMN, SIZE_COLUMN | COLUMN_COMMAS | COLUMN_AUTO, DATE_COLUMN}, {0, 0, 0}, 3,
+		/* 00 */ {{COLUMN_MARK | NAME_COLUMN, SIZE_COLUMN | COLUMN_COMMAS | COLUMN_AUTO, DATE_COLUMN}, {0, 6, 0}, 3,
 				{COLUMN_RIGHTALIGN | NAME_COLUMN}, {0}, 1,  0, 1, 0, 0, 0, 0,
 				{COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH}, {COUNT_WIDTH}},
 
 		/* 01 */ {{NAME_COLUMN, NAME_COLUMN, NAME_COLUMN}, {0, 0, 0}, 3,
 				{COLUMN_RIGHTALIGN | NAME_COLUMN, OWNER_COLUMN, GROUP_COLUMN, SIZE_COLUMN, DATE_COLUMN, TIME_COLUMN},
-				{0, 6, 6, 0, 0, 5}, 6, 0, 1, 0, 0, 0, 0, {COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH},
+				{0, 6, 6, 6, 0, 5}, 6, 0, 1, 0, 0, 0, 0, {COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH},
 				{COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH}},
 
 		/* 02 */ {{NAME_COLUMN, NAME_COLUMN}, {0, 0}, 2,
 				{COLUMN_RIGHTALIGN | NAME_COLUMN, OWNER_COLUMN, GROUP_COLUMN, SIZE_COLUMN, DATE_COLUMN, TIME_COLUMN},
-				{0, 6, 6, 9, 0, 5}, 6, 0, 0, 0, 0, 0, 0, {COUNT_WIDTH, COUNT_WIDTH},
+				{0, 6, 6, 6, 0, 5}, 6, 0, 0, 0, 0, 0, 0, {COUNT_WIDTH, COUNT_WIDTH},
 				{COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH}},
 
-		/* 03 */ {{NAME_COLUMN, SIZE_COLUMN | COLUMN_AUTO, DATE_COLUMN, TIME_COLUMN}, {0, 0, 0, 5}, 4,
+		/* 03 */ {{NAME_COLUMN, SIZE_COLUMN | COLUMN_AUTO, DATE_COLUMN, TIME_COLUMN}, {0, 6, 0, 5}, 4,
 				{COLUMN_RIGHTALIGN | NAME_COLUMN, OWNER_COLUMN, GROUP_COLUMN}, {0, 6, 6}, 3, 0, 1, 0, 0, 0, 0,
 				{COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH}, {COUNT_WIDTH}},
 
-		/* 04 */ {{NAME_COLUMN, SIZE_COLUMN | COLUMN_COMMAS | COLUMN_AUTO}, {0, 0}, 2,
+		/* 04 */ {{NAME_COLUMN, SIZE_COLUMN | COLUMN_AUTO}, {0, 6}, 2,
 				{COLUMN_RIGHTALIGN | NAME_COLUMN, SIZE_COLUMN, OWNER_COLUMN, GROUP_COLUMN, DATE_COLUMN, TIME_COLUMN},
-				{0, 9, 6, 6, 0, 5}, 6, 0, 0, 0, 0, 0, 0, {COUNT_WIDTH, COUNT_WIDTH},
+				{0, 6, 6, 6, 0, 5}, 6, 0, 0, 0, 0, 0, 0, {COUNT_WIDTH, COUNT_WIDTH},
 				{COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH}},
 
 		/* 05 */ {{NAME_COLUMN, SIZE_COLUMN, PHYSICAL_COLUMN, WDATE_COLUMN, CDATE_COLUMN, ADATE_COLUMN, OWNER_COLUMN, GROUP_COLUMN, ATTR_COLUMN},
-				{0, 0, 0, 14, 14, 14, 6, 6, 0}, 9, {COLUMN_RIGHTALIGN | NAME_COLUMN}, {0}, 1, 1, 1, 0, 0, 0,
+				{0, 6, 6, 14, 14, 14, 6, 6, 0}, 9, {COLUMN_RIGHTALIGN | NAME_COLUMN}, {0}, 1, 1, 1, 0, 0, 0,
 				0,
 				{COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH},
 				{COUNT_WIDTH}},
 
 		/* 06 */ {{NAME_COLUMN, DIZ_COLUMN}, {40, 0}, 2,
 				{COLUMN_RIGHTALIGN | NAME_COLUMN, OWNER_COLUMN, GROUP_COLUMN, SIZE_COLUMN, DATE_COLUMN, TIME_COLUMN},
-				{0, 6, 6, 9, 0, 5}, 6, 0, 1, 0, 0, 0, 0, {PERCENT_WIDTH, COUNT_WIDTH},
+				{0, 6, 6, 6, 0, 5}, 6, 0, 1, 0, 0, 0, 0, {PERCENT_WIDTH, COUNT_WIDTH},
 				{COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH}},
 
-		/* 07 */ {{NAME_COLUMN, SIZE_COLUMN | COLUMN_AUTO, DIZ_COLUMN}, {0, 0, 54}, 3, {COLUMN_RIGHTALIGN | NAME_COLUMN}, {0}, 1, 1,
+		/* 07 */ {{NAME_COLUMN, SIZE_COLUMN | COLUMN_AUTO, DIZ_COLUMN}, {0, 6, 54}, 3, {COLUMN_RIGHTALIGN | NAME_COLUMN}, {0}, 1, 1,
 				1, 0, 0, 0, 0, {COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH}, {COUNT_WIDTH}},
 
-		/* 08 */ {{NAME_COLUMN, SIZE_COLUMN, OWNER_COLUMN, GROUP_COLUMN}, {0, 0, 6, 15}, 4,
+		/* 08 */ {{NAME_COLUMN, SIZE_COLUMN, OWNER_COLUMN, GROUP_COLUMN}, {0, 6, 6, 15}, 4,
 				{COLUMN_RIGHTALIGN | NAME_COLUMN, OWNER_COLUMN, GROUP_COLUMN, SIZE_COLUMN, DATE_COLUMN, TIME_COLUMN},
-				{0, 6, 6, 9, 0, 5}, 6, 0, 1, 0, 0, 0, 0, {COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH},
+				{0, 6, 6, 6, 0, 5}, 6, 0, 1, 0, 0, 0, 0, {COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH},
 				{COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH}},
 
-		/* 09 */ {{NAME_COLUMN, SIZE_COLUMN | COLUMN_AUTO, NUMLINK_COLUMN}, {0, 0, 4}, 3,
+		/* 09 */ {{NAME_COLUMN, SIZE_COLUMN | COLUMN_AUTO, NUMLINK_COLUMN}, {0, 6, 3}, 3,
 				{COLUMN_RIGHTALIGN | NAME_COLUMN, OWNER_COLUMN, GROUP_COLUMN, SIZE_COLUMN, DATE_COLUMN, TIME_COLUMN},
-				{0, 6, 6, 9, 0, 5}, 6, 0, 1, 0, 0, 0, 0, {COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH},
+				{0, 6, 6, 6, 0, 5}, 6, 0, 1, 0, 0, 0, 0, {COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH},
 				{COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH, COUNT_WIDTH}}};
 
 size_t SizeViewSettingsArray = ARRAYSIZE(ViewSettingsArray);
